extensions/wikia/WikiFactory/Close/maintenance.php

```diff
@@ -202,87 +202,103 @@ public function execute() {
 			}
 			if ( $row->city_flags & WikiFactory::FLAG_DELETE_DB_IMAGES ||
 				 $row->city_flags & WikiFactory::FLAG_FREE_WIKI_URL ) {
+				$where["city_id"] = $cityid;
```

Comment:
Please add a comment here, maybe with the ticket number. The `$where` array is created way up in the code and it is almost impossible for the reader to understand what we're checking here.

Reply:
It's added, but you'll see it in file explorer